src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -809,13 +733,24 @@ abstract class ManagedStrategy(
             _trace += StateRepresentationTracePoint(iThread, currentActorId[iThread], stateRepresentation, callStackTrace)
         }
     }
+
+
+    // == NVM related ==
+
+    internal fun beforeCrashPoint(iThread: Int) = newCrashPoint(iThread)
+    internal fun beforeNVMOperation(iThread: Int, codeLocation: Int, tracePoint: MethodCallTracePoint?) =
```

Comment:
Could you please add comments for these methods?
In some sense they are internal api for transformers, so it is useful to know, for example, whether `NVM operation` is just read/write from a shared location or it can be something else